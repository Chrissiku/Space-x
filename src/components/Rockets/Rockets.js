/* eslint-disable linebreak-style */
/* eslint-disable no-unused-vars */
/* eslint-disable quotes */

const Rockets = () => (
  <>
    <h1>Rockets page</h1>
  </>
);

export default Rockets;
