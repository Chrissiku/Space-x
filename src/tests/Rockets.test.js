/* eslint-disable linebreak-style */
/* eslint-disable comma-dangle */
/* eslint-disable quotes */
import React from "react";
import TestRenderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../Redux/ConfigureStore";
import Rockets from "../components/Rockets/Rockets";

it("Rockets render correctly", () => {
  const Tree = TestRenderer.create(
    <Provider store={store}>
      <Rockets />
    </Provider>
  );
  expect(Tree).toMatchSnapshot();
});
