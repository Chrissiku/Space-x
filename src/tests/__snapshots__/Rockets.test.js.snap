// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets render correctly 1`] = `
<div
  className="rockets-container container"
/>
`;
